libcxx/include/map

```diff
@@ -57,177 +57,209 @@ public:
+    constexpr const_reverse_iterator rbegin() const noexcept; // constexpr since C++26
+          constexpr reverse_iterator rend() noexcept; // constexpr since C++26
+    constexpr const_reverse_iterator rend()   const noexcept; // constexpr since C++26
 
-    const_iterator         cbegin()  const noexcept;
-    const_iterator         cend()    const noexcept;
-    const_reverse_iterator crbegin() const noexcept;
-    const_reverse_iterator crend()   const noexcept;
+    constexpr const_iterator         cbegin()  const noexcept; // constexpr since C++26
+    constexpr const_iterator         cend()    const noexcept; // constexpr since C++26
+    constexpr const_reverse_iterator crbegin() const noexcept; // constexpr since C++26
+    constexpr const_reverse_iterator crend()   const noexcept; // constexpr since C++26
 
     // capacity:
-    bool      empty()    const noexcept;
-    size_type size()     const noexcept;
-    size_type max_size() const noexcept;
+    constexpr bool      empty()    const noexcept; // constexpr since C++26
+    constexpr size_type size()     const noexcept; // constexpr since C++26
+    constexpr size_type max_size() const noexcept; // constexpr since C++26
 
     // element access:
-    mapped_type& operator[](const key_type& k);
-    mapped_type& operator[](key_type&& k);
+    constexpr mapped_type& operator[](const key_type& k); // constexpr since C++26
+    constexpr mapped_type& operator[](key_type&& k); // constexpr since C++26
 
-          mapped_type& at(const key_type& k);
-    const mapped_type& at(const key_type& k) const;
+          constexpr mapped_type& at(const key_type& k); // constexpr since C++26
+    constexpr const mapped_type& at(const key_type& k) const; // constexpr since C++26
 
     // modifiers:
     template <class... Args>
-        pair<iterator, bool> emplace(Args&&... args);
+        constexpr pair<iterator, bool> emplace(Args&&... args); // constexpr since C++26
     template <class... Args>
-        iterator emplace_hint(const_iterator position, Args&&... args);
-    pair<iterator, bool> insert(const value_type& v);
-    pair<iterator, bool> insert(      value_type&& v);                                // C++17
-    template <class P>
-        pair<iterator, bool> insert(P&& p);
-    iterator insert(const_iterator position, const value_type& v);
-    iterator insert(const_iterator position,       value_type&& v);                   // C++17
-    template <class P>
-        iterator insert(const_iterator position, P&& p);
-    template <class InputIterator>
-        void insert(InputIterator first, InputIterator last);
+        constexpr iterator emplace_hint(const_iterator position, Args&&... args); // constexpr since C++26
+    constexpr pair<iterator, bool> insert(const value_type& v); // constexpr since C++26
+    constexpr pair<iterator, bool> insert(      value_type&& v);                                // C++17, constexpr
+since C++26
```

Comment:
```suggestion
        constexpr iterator emplace_hint(const_iterator position, Args&&... args); // constexpr since C++26
    constexpr pair<iterator, bool> insert(const value_type& v);                   // constexpr since C++26
    constexpr pair<iterator, bool> insert(      value_type&& v);                  // C++17, constexpr since C++26
```
Can you align these comments a bit?

Reply:
Done in 
66b4d6f2b0aa

Aligned it to each "stanza" / section, let me know if this looks good